src/EnergyPlus/OutputReportTabularAnnual.cc

```diff
@@ -624,16 +624,32 @@ namespace OutputReportTabularAnnual {
 
     void WriteAnnualTables(EnergyPlusData &state)
     {
-        // Jason Glazer, August 2015
-        // This function is not part of the class but acts as an interface between procedural code and the class by
-        // invoking the writeTable member function for each of the AnnualTable objects
-        std::vector<AnnualTable>::iterator annualTableIt;
-        for (annualTableIt = annualTables.begin(); annualTableIt != annualTables.end(); ++annualTableIt) {
-            annualTableIt->writeTable(state, state.dataOutRptTab->unitsStyle);
+        for (int iUnitSystem = 0; iUnitSystem <= 1; iUnitSystem++) {
+            OutputReportTabular::iUnitsStyle unitsStyle_cur = state.dataOutRptTab->unitsStyle;
+            bool produceTabular = true;
+            bool produceSQLite = false;
+            if (produceDualUnitsFlags(iUnitSystem,
+                                      state.dataOutRptTab->unitsStyle,
+                                      state.dataOutRptTab->unitsStyle_SQLite,
+                                      unitsStyle_cur,
+                                      produceTabular,
+                                      produceSQLite))
+                break;
+
+            // Jason Glazer, August 2015
+            // This function is not part of the class but acts as an interface between procedural code and the class by
+            // invoking the writeTable member function for each of the AnnualTable objects
+            std::vector<AnnualTable>::iterator annualTableIt;
+            for (annualTableIt = annualTables.begin(); annualTableIt != annualTables.end(); ++annualTableIt) {
+                // annualTableIt->writeTable(state, state.dataOutRptTab->unitsStyle);
+                // Jan 2021: Use the revised version with added parameters for dual-unit reporting
+                annualTableIt->writeTable(state, unitsStyle_cur, produceTabular, produceSQLite);
+            }
         }
     }
 
-    void AnnualTable::writeTable(EnergyPlusData &state, OutputReportTabular::iUnitsStyle unitsStyle)
+    // Jan 2021: This function has been modifed to accommondate more parameters for dual-unit reporting
```

Comment:
I don't think we need commentary like this.  Git blame can show us exactly when this changed along with the context of the change.  I'm not opposed to comments in the code, but if they are needed, just expand on the impact of the changes in the function rather than saying the function was modified.

Reply:
Comments removed.